share/mdnjs/parse.sh

```diff
@@ -1,5 +1,15 @@
 #!/bin/bash
-python parse.py --out output.txt \
+rm output.txt
+
+python parse.py --out output.api.txt \
```

Comment:
hmm well the issue is that the Web API docs took several hours to download so I saved them in a separate dir and parsed them separately.  I was thinking we would want to update either the API or JS docs separately but I'm reconsidering that choice.